charts/images.yaml

```diff
@@ -57,7 +62,7 @@ images:
 - name: csi-attacher
   sourceRepository: github.com/kubernetes-csi/external-attacher
   repository: k8s.gcr.io/sig-storage/csi-attacher
-  tag: "v2.2.0"
+  tag: "v3.3.0"
```

Comment:
Is this version compatible with 1.18-1.21? :)

Reply:
Yep :) https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.3.0 see "Minimum Kubernetes version". I also tested it.